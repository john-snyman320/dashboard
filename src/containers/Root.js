import React from 'react';
import { 
  BrowserRouter as Router
} from 'react-router-dom';

import createHistory from 'history/createBrowserHistory';
import { Provider } from 'react-redux';

import store from '../store';

import App from './App';

const history = createHistory();

class Root extends React.Component {
  render()  {
    return (
      <Provider store={store}>
        <Router history={history}>   
          <App />
        </Router>
      </Provider>
      
    );
  }
}

export default Root;