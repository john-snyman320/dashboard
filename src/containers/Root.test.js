import React from 'react';
import renderer from 'react-test-renderer';
import Root from '../containers/Root';

describe('Root container', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <Root />
    ).toJSON();
    
    expect(tree).toMatchSnapshot();
  });
});