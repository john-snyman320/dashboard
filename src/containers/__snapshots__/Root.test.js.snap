// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root container renders correctly 1`] = `
<div
  className="ui loader"
/>
`;
